src/gc/impl/conservative/gc.d

```diff
@@ -1244,9 +1240,8 @@ class ConservativeGC : GC
 
         usize = bsize - flsize;
 
-        stats.poolsize = psize;
-        stats.usedsize = bsize - flsize;
-        stats.freelistsize = flsize;
+        stats.heapsize = bsize - flsize;
```

Comment:
That's misleading, you're computing used size here.
Either rename the stats parameter or change the computation.